### ISSUE TITLE
Fall sounds are block break sounds

### ISSUE THREAD

**Issue by akhilmitta** [GeyserMC/Geyser#1173]
**Describe the bug**
When I try to fall down from creative fly for an example to the ground, the game doesn't produce a fall-down sound, it produces a block break sound. Try it while jumping from a hill to the ground as well, instead of having that fall sound, you'll have the sound of a block breaking based on what block you land

**To Reproduce**
Go into creative mode and fly up about 10 blocks or so ( enough to make fall damage occur )
Stop your flight and let gravity do it's thing, fall down and you will be hearing a sound of the block breaking, not the fall damage sound
You can try this on glass too and you will notice the glass break sound will occur.

**Expected behavior**
The sound of a block break instead of the sound of falling from a great height. This happens only once in a while, so it's not that much of an issue, but it still occurs.

**Screenshots / Videos**

**Server Version**
This server is running Paper version git-Paper-135 (MC:1.16.1) ( implementing API version 1.16.1-R0.1-SNAPSHOT)

**Geyser Version**
This server is running Geyser version 1.1.0 (git-master-d6290cc) (Java: 1.16.2, Bedrock: 1.16.20)

**Minecraft: Bedrock Edition Version**
Bedrock 1.16.20

**Additional Context**

**Comment by DoctorMacc** [comment]
Thanks for the issue! However, this is a duplicate of #707.
